### DISCUSSION TITLE
order of overlays in menu

### DISCUSSION THREAD

**Discussion by legofahrrad** [streetcomplete/StreetComplete#4673] [Ideas]
As there follow 1-2 new overlays in every major update, it would make sense to order the overlays in the menu where you choose them / switch between them. Putting new overlays at the bottom of the list isn't a good idea I guess.
In my opinion they should be ordered or categorised in "node editing" and "way editing".

| **Node editing** | **Way editing** |
| --- | --- |
| Addresses | Street lighting |
| Shops | Sidewalks |
|  | Cycleways |
|  | Street parking |

<hr>

![Screenshot 2022-12-06 at 20-17-45 Foto - Google Fotos](https://user-images.githubusercontent.com/92865620/206002113-cced9d7e-0405-4972-99d4-ff4c9ec4064d.png)
<br>
The current menu can get messy over the time, especially when there are more overlays coming in the future.

**Comment by westnordost** [comment]
ok 🤷 , currently there is no order at all, so some kind of order is better than none I guess.

Hm no, on the other hand, add a few overlays and the whole dialog needs to be restructured (to be full screen etc) anyway, maybe allowing for a short description too. So for now , I will not change anything here.
